mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorPasses.cpp

```diff
@@ -214,6 +214,8 @@ struct SparseTensorCodegenPass
     target.addLegalOp<GetStorageSpecifierOp>();
     target.addLegalOp<SetStorageSpecifierOp>();
     target.addLegalOp<StorageSpecifierInitOp>();
+    // tensor::FromElementsOp might be yield after lowering unpack.
```

Comment:
At first glance, this looks like commented out code ;-)
So please change to

// Note that `tensor::FromElementsOp` might be yield after lowering unpack.